src/EditorFeatures/Text/TextEditorFeatures.csproj

```diff
@@ -58,6 +58,8 @@
     <InternalsVisibleToTest Include="Roslyn.Services.Editor.VisualBasic.UnitTests" />
     <InternalsVisibleToTest Include="Roslyn.VisualStudio.Services.UnitTests" />
     <InternalsVisibleToTest Include="RoslynETAHost" />
+    <InternalsVisibleToFSharp Include="FSharp.Editor" />
```

Comment:
You might want to consider naming these the same way we do for C#/VB/TS.  I think uniformity will make things a lot clearer.